setup.py

```diff
@@ -118,6 +118,7 @@ def get_git_sha():
         "elasticsearch": ["elasticsearch-dbapi>=0.1.0, <0.2.0"],
         "druid": ["pydruid==0.5.7", "requests==2.22.0"],
         "hana": ["hdbcli==2.4.162", "sqlalchemy_hana==0.4.0"],
+        "dremio": ["sqlalchemy_dremio"]
```

Comment:
We usually set the minimum version at the current most stable version (`>=0.5.0dev0`).